package-lock.json

```diff
@@ -1,37 +1,1836 @@
 {
   "name": "prerender-node",
   "version": "3.3.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
Please stay at lock file version 1 for now.
Use `nvm use --lts` before running `npm install`.

Reply:
I think it is not necessary since v2 is backward compatible with v1.

[https://github.blog/2021-02-02-npm-7-is-now-generally-available/#changes-to-the-lockfile](https://github.blog/2021-02-02-npm-7-is-now-generally-available/#changes-to-the-lockfile)